lib/world_cup.rb

```diff
@@ -0,0 +1,55 @@
+class WorldCup
+  attr_reader :year
+
+  def initialize(year, teams)
+    @year = year
+    @teams = teams
+  end
+
+  def teams
+    @teams.find_all do |team|
+      team.eliminated == false
+    end
+  end
+
+  def active_players_by_position(position)
```

Comment:
This looks AWESOME - I really love that you split out an active teams method above!!